lib/Twig/Extension/Core.php

```diff
@@ -1463,15 +1463,25 @@ function twig_get_attribute(Twig_Environment $env, Twig_Source $source, $object,
         throw new Twig_Error_Runtime('Accessing Twig_Template attributes is forbidden.');
     }
 
+    static $extensionSandbox = array();
+    $envHash = spl_object_hash($env);
```

Comment:
in terms of the spl_object_hash - does it actually makes more sense to look at the value of $this->optionsHash on the environment? [that would appear to get updated whenever a new extension is added]